Client/Ecosia/UI/Onboarding/WelcomeTourGreen.swift

```diff
@@ -5,18 +5,44 @@
 import UIKit
 
 final class WelcomeTourGreen: UIView, NotificationThemeable {
-    private weak var searchLabel: UILabel!
+    private lazy var searchLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = .localized(.sustainableShoes)
+        label.font = .systemFont(ofSize: 15)
+        label.numberOfLines = 1
+        label.textAlignment = .left
+        return label
+    }()
+    private lazy var counterLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "10"
+        label.font = .systemFont(ofSize: 17).bold()
+        label.numberOfLines = 1
+        label.textAlignment = .left
+        return label
+    }()
+    private lazy var counterSubtitleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = .localizedPlural(.searches, num: 500)
+        label.font = .systemFont(ofSize: 13)
+        label.numberOfLines = 1
+        label.textAlignment = .left
+        return label
+    }()
 
-    init() {
+    init(hasCounter: Bool = false) {
```

Comment:
I'm OK with adding it here and don't have strong pushback.
Sharing my 💭 that personally, I would have made it like the below 👇 : 
```suggestion
    init(isCounterEnabled: Bool = false) {
```

Modifying the `setup(:) as well

Reply:
Also like your suggestion, will change it 🙂